Language/Structure/Further Syntax/define.adoc

```diff
@@ -24,7 +24,7 @@ subCategories: [ "Further Syntax" ]
 This can have some unwanted side effects though, if for example, a constant name that had been #defined is included in some other constant or variable name. In that case the text would be replaced by the #defined number (or text).
 [%hardbreaks]
 
-In general, the `link:../../../variables/variable-scope\--qualifiers/const[const]` keyword is preferred for defining constants and should be used instead of `#define`.
+In general, the `link:../../variables/variable-scope-qualifiers/const[const]` keyword is preferred for defining constants and should be used instead of `#define`.
```

Comment:
```suggestion
In general, the `link:../../../variables/variable-scope-qualifiers/const[const]` keyword is preferred for defining constants and should be used instead of `#define`.
```
The extra level of `../` is required. The confusing thing is that here on GitHub, each page is a file, but on arduino.cc each file is a folder.

The extra `-` definitely needs to go though, good catch!